src/style.css

```diff
@@ -0,0 +1,132 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  --search-icon: url("./images/Search.svg");
+  --white-color: #fff;
+  --border-color: #e1e7ed;
+  --border-focus-color: #f6f6f7;
+}
+
+.search-bar__label {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+}
+
+.search-bar__form {
+  margin-top: 20px;
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
+  background: var(--white-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__icon--big {
+  background-image: var(--search-icon);
+  background-size: 19px;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.search-bar__form--main {
+  height: 68px;
+  position: relative;
+}
+
+.search-bar__input--big {
+  margin: 25px auto 26px 62px;
+  width: 100%;
+  border: none;
+  outline: none;
+}
+
+.search-bar__input--big::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.search-bar__input--big:focus {
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  font-family: "Avenir-Heavy", sans-serif;
```

Comment:
Move this property to html tag and here only inherit it. fix all cases. For changing fonts use `font-weight` property in `@font-face` and then in classes use right font-weight value